src/concrete/ml/sklearn/base.py

```diff
@@ -2069,22 +2105,53 @@ def post_processing(self, y_preds: numpy.ndarray) -> numpy.ndarray:
         Returns:
             numpy.ndarray: The majority vote.
         """
-        y_preds_processed = []
-        for y in y_preds:
-            vote = self.majority_vote(y.flatten())
-            y_preds_processed.append(vote)
+        return numpy.array([self.majority_vote(y.flatten()) for y in y_preds])
 
-        return numpy.array(y_preds_processed)
+    def get_topk_labels(self, X: Data, fhe: Union[FheMode, str] = FheMode.DISABLE) -> numpy.ndarray:
+        """Return the K-nearest labels of a point.
 
-    def predict(self, X: Data, fhe: Union[FheMode, str] = FheMode.DISABLE) -> numpy.ndarray:
+        Args:
+            X (Data): The input values to predict, as a Numpy array, Torch tensor, Pandas DataFrame
```

Comment:
"The input values to predict" -> "The input values to consider" or something like that I think

Reply:
it was a copy past, do you want me to change the original one ?

@RomanBredehoft

Reply:
yes but because the original one was for "predict", here it's not really the predict method so it feels less relevant to keep it that way

Reply:
not very important anyway